TERMS-OF-USE.md

```diff
@@ -1,4 +1,4 @@
-# **TERMS OF USE**  **FOR MOBILECOINS AND MOBILECOIN WALLETS**
+# **TERMS OF USE** **FOR MOBILECOINS AND MOBILECOIN WALLETS**
```

Comment:
```suggestion
# **TERMS OF USE**  **FOR MOBILECOINS AND MOBILECOIN WALLETS**
```

Reply:
this change will make the file equal to the other ones which have 'md5' hash:

`TERMS-OF-USE.md) = 6fc7a4a7a21915add9f36fd20da4e8d3`